horace_core/sqw/coord_transform/@aProjectionBase/aProjectionBase.m

```diff
@@ -171,6 +178,24 @@
             end
             [obj,par] = init(obj,varargin{:});
         end
+        function obj = copy_proj_param_from_source(obj,cut_source)
+            % a projection may need some information from the object,
+            % it used to process. For example, all projections used for cut
+            % need to know lattice parameters of sqw object used as source
```

Comment:
```suggestion
            % Load information into the projection from the object being processed, 
            % which is required to perform the operation. For example, all projections 
            % used for cut need to know lattice parameters of sqw object used as source
```
the original text says why you need the information but not what the method actually does with the information.